src/Stormpath/Resource/Resource.php

```diff
@@ -32,6 +32,7 @@ class Resource extends Magic
     private $options;
 
     const HREF_PROP_NAME = "href";
+    const CUSTOMDATA_PROP_NAME = "customData";
```

Comment:
I would rather add this constant to the `CustomData` class and use it from where ever you need to use it.